docs/en/userguide/faq.md

```diff
@@ -119,4 +119,9 @@ Modify: 2021-01-19 00:16:21.539559082 +0800
 Change: 2021-01-19 00:16:21.539559082 +0800
  Birth: -
 
-```
\ No newline at end of file
+```
+
+## 7. Why does volume attachment timeout occur when PVC is used in an application?
+**Answer**: First, you need to use the command `kubectl get csidriver` to check whether CSI driver is installed.
```

Comment:
the command should be `export KUBECONFIG=/etc/kubernetes/kubelet.conf && kubectl get csidriver`